packages/client/modules/meeting/components/ScaleDropdownMenuItem.tsx

```diff
@@ -5,20 +5,21 @@ import {createFragmentContainer} from 'react-relay'
 import textOverflow from '~/styles/helpers/textOverflow'
 import {PALETTE} from '~/styles/paletteV2'
 import {FONT_FAMILY} from '~/styles/typographyV2'
-import {PokerCards} from '../../../types/constEnums'
+import Icon from '../../../components/Icon'
 import MenuItem from '../../../components/MenuItem'
 import useAtmosphere from '../../../hooks/useAtmosphere'
-import {MenuProps} from '../../../hooks/useMenu'
 import useMutationProps from '../../../hooks/useMutationProps'
 import UpdatePokerTemplateDimensionScaleMutation from '../../../mutations/UpdatePokerTemplateDimensionScaleMutation'
+import {PokerCards} from '../../../types/constEnums'
 import {ScaleDropdownMenuItem_dimension} from '../../../__generated__/ScaleDropdownMenuItem_dimension.graphql'
 import {ScaleDropdownMenuItem_scale} from '../../../__generated__/ScaleDropdownMenuItem_scale.graphql'
 import ScaleActions from './ScaleActions'
 
 interface Props {
   scale: ScaleDropdownMenuItem_scale
+  scaleCount: number
   dimension: ScaleDropdownMenuItem_dimension
-  menuProps: MenuProps
+  closePortal: () => void
```

Comment:
never give a component more than it needs. i try to make a game of passing as few props as possible & if the prop is an object, as small an object as possible